mission_list.md

```diff
@@ -0,0 +1,60 @@
+# Mission List
+
+Quote form the [constitution](https://github.com/polkadot-tooling-collective/constitution):
+
+>The Mission List contains all projects and services that are maintained by PoToC.
+>
+>It is divided into two sections, solely to keep the requirements of each section clearly defined; Core and dApp.
+>It can be updated through an on-chain 2/3-majority rank-weighted vote across all Members.
+>
+>Some obvious cornerstones that must be upheld by all projects are:
+>
+>- Must be FOSS (Free and Open-Source Software).
+>- Source code must be accessible without a login requirement.
+>- All members of PoToC must have permission to propose changes and raise issues.
+>- The Mission List will be defined as part of the initial seeding.
+>
+>Core Tooling
+>
+>Generally, all tools that are used by The Fellowship to craft, test, and validate updates fall into this category. But only the tools that are predominantly developed for this cause shall be maintained by PoToC. For example, the Rust language is not predominantly developed for Polkadot; hence, it does not fit here.
+>
+>dApp Tooling
+>
+>This category contains all tools and libraries that are used to build on Polkadot and are predominantly developed for this cause.
+
+(This is my personal opinion on a 
+
+## DApp Tooling
+
+- [Polkadot-JS](https://github.com/polkadot-js/apps)
+- [PAPI](https://www.npmjs.com/package/@polkadot-api/substrate-client)
+- [Chopsticks](https://github.com/AcalaNetwork/chopsticks)
+- [Zombienet](https://github.com/paritytech/zombienet) + [v2](https://github.com/paritytech/zombienet-sdk)
+- [Substrate Connect](https://github.com/paritytech/substrate-connect)
+- [Scale TS](https://npmjs.com/package/scale-ts)
+
+## Core Tooling
+
+- [SubXt](https://github.com/paritytech/subxt)
+- [Subway](https://github.com/AcalaNetwork/subway)
+- [Zepter](https://github.com/ggwpez/zepter)
+- [Substrate Weight Compare](https://github.com/ggwpez/substrate-weight-compare)
```

Comment:
Please call me out if you think that my tools are not belonging to the list, i am biased in this regard 😄